Wikimate.php

```diff
@@ -707,6 +708,44 @@ public function delete($array)
         return $this->request($array, $headers, true);
     }
 
+    /**
+     * Performs an undelete query to the wiki API.
+     *
+     * @param   string         $name     The original name of the wiki article or file
+     * @param   string         $reason   Reason for the restore
+     * @param   boolean        $talktoo  True to restore talk page, if any
```

Comment:
Just a tiny nitpick :smile:

```suggestion
     * @param   boolean        $talktoo  True to restore talk page, if any;
```